.theme/store/piniaStore.js

```diff
@@ -0,0 +1,12 @@
+import { defineStore } from "pinia";
+
+export const usePiniaStore = defineStore("piniaStore", {
+  state: () => ({
+    titles: {}
```

Comment:
titles is going to be an array, but it is initialized as an object which is confusing. make it null or an empty array